pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -529,8 +529,7 @@ def _should_check_val_fx(self, batch_idx: int, is_last_batch: bool) -> bool:
     def _save_loggers_on_train_batch_end(self) -> None:
         """Flushes loggers to disk."""
         # this assumes that `batches_that_stepped` was increased before
-        should_flush = self._batches_that_stepped % self.trainer.flush_logs_every_n_steps == 0
```

Comment:
@awaelchli Since I have removed the `trainer.flush_logs_every_n_steps` attribute, I have made this change. Do you think this is suitable?

Reply:
Yes, this is probably the best. We let the logger decide when to flush, so the training loop should just call the logger method.